### PR TITLE
Seedlet: Fix full width Video block

### LINKED ISSUE #2553: Seedlet: Full width video is aligned to the left instead
## Steps to replicate

Embed a video using the Wordpress embed blocks. Align it to full width. Go to the frontend.

## Result

Editor:

Screenshot 2020-10-06 at 09 47 44

Frontend:

Screenshot 2020-10-06 at 09 47 22

## Expected

I expected the frontend to look like the editor and for the video to be full width.

### LINKED ISSUE #3032: Seedlet: Full-width YouTube embeds do not appear full-width in the front end.
Potentially a problem with [responsive embeds](https://developer.wordpress.org/block-editor/developers/themes/theme-support/#responsive-embedded-content)? 

## Steps to replicate

1. Add a YouTube embed. 
2. Make it Full-width.
3. Note that it looks correct in the editor.
4. View the post on the front end, note that the video is small and in the upper left corner of where it should be. 

## Screenshots

Editor

Screen Shot 2021-01-14 at 5 36 08 PM

Front End

Screen Shot 2021-01-14 at 5 35 52 PM

### PR DESCRIPTION
**Pull request by danieldudzic**
Closes #2553 

Before:
video
After:
video_after

### CODE REVIEW

File: seedlet/inc/jetpack.php [hunk 1]
```diff
@@ -14,9 +14,6 @@
  * See: https://jetpack.com/support/responsive-videos/
  */
 function seedlet_jetpack_setup() {
-	// Add theme support for Responsive Videos.
-	add_theme_support( 'jetpack-responsive-videos' );
```
Comment (scruffian) [inline seedlet/inc/jetpack.php:18]: What other impact does removing this line have? Is there a reason we want to remove support for responsive videos? (https://jetpack.com/support/responsive-videos/)

Comment (danieldudzic) [reply seedlet/inc/jetpack.php:18]: Varia is missing the `jetpack-responsive-videos` support and videos seem to be sized correctly and maintain their responsiveness.

I'm assuming this was needed in older themes that didn't account for responsive videos.

Comment (scruffian) [reply seedlet/inc/jetpack.php:18]: Yeah I did a bit more digging and found this: https://github.com/Automattic/jetpack/issues/10678#issuecomment-497023187

Looks like its safe to remove.

**Review by scruffian — APPROVED** [review]
LGTM

File: seedlet/inc/jetpack.php (hunk 1 above)
Comment (danieldudzic) [reply seedlet/inc/jetpack.php:18]: Great, thanks for double-checking. I will also remove it from TT1.

**Pull request merged**